src/browser/deck/actions.js

```diff
@@ -214,24 +273,19 @@ define((require, exports, module) => {
   // on a result of this function as they assume to have an active item.
   const deactivate = items => items.update(indexOfActive(items), asInactive);
 
-  const insert = (items, item, index) => {
-    const include = items =>
-      index <= 0 ? items.push(asFirst(items, item)) :
-      index >= items.count() - 1 ? items.push(asLast(items, item)) :
-      items.push(asNth(items, index, item));
-
-    const update = compose(include,
-                           isSelected(item) ? deselect : identity,
-                           isActive(item) ? deactivate : identity);
-
-    return update(items);
+  const include = (items, item) => {
+    const edit = compose(items => items.push(item),
```

Comment:
Possible bug? `Array.prototype.push` returns [new length property of the object upon which the method was called](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push). Since you are composing, I would expect you need to return `item`.